UnityProject/Assets/Scripts/Core/SecureStuff/AllowedReflection.cs

```diff
@@ -15,6 +15,13 @@ public class BaseAttribute : Attribute
 	{
 	}
 
+	/// <summary>
+	/// Put this on Fields that Reference other objects, where you don't want funny scenarios where the map saver overwrites the networked state
+	/// </summary>
+	public class IsSyncedAttribute : Attribute
```

Comment:
this can get confused with game object `Attribute` which is used in `ObjectAttributes` and `ItemAttributes`

Reply:
what? though I've noticed a different error with it

Reply:
are you using the base attribute class of `ItemAttributes` or something else? if its something else, change its name to be a bit more distinctive

Reply:
nah, this is just symbolising an attribute so you can slap on stuff to say it synchronised so you don't need to apply it when you're the client